### PR TITLE
Translation to Italian of Rust tutorial

### PR DESCRIPTION
**Pull request by carlomilanesi**

### CODE REVIEW

**Comment by carlomilanesi** [comment]
Created Italian translation of tutorial about Rust, taken from file https://github.com/adambard/learnxinyminutes-docs/blob/master/rust.html.markdown

**Comment by vendethiel** [comment]
Why close?

**Comment by carlomilanesi** [comment]
On 05/29/2016 03:15 AM, ven wrote:
> Why close?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/adambard/learnxinyminutes-docs/pull/2269#issuecomment-222337164>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIZqmFF61qYp41WW3xU4OpJj55vg_kAiks5qGOi5gaJpZM4IpE0i>.

I don't know how to use GitHub. You asked me for a Pull Request, and so 
I read the GitHub tutorial to discover how to do Pull Requests, and I 
did it, but it came out "close".
Surely I did something wrong.
However I tried to add two requests:
- A small change of the English Rust tutorial.
- A translation into Italian of the whole English Rust tutorial.

--
Carlo Milanesi

File: it-it/rust-it.html.markdown [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+    for i in array.iter() {
+        println!("{}", i);
+    }
+
+    // Range
+    for i in 0u32..10 {
+        print!("{} ", i);
+    }
+    println!("");
+    // prints `0 1 2 3 4 5 6 7 8 9 `
```
Comment (vendethiel) [inline it-it/rust-it.html.markdown]: Is "prints" correct here?

Comment (carlomilanesi) [inline it-it/rust-it.html.markdown]: No. It remained the original English text.
Please put "Stampa" instead of "prints".

Comment (vendethiel) [inline it-it/rust-it.html.markdown]: I can do it if you want, but to explain how you can do it yourself:

- edit the file as you would
- add the file to the list of changes: `git add it-it/rust-it.html.markdown`
- make a new commit: `git commit -m "Fixing a word still in english"`
- push to update the github version: `git push`.

Your changes will appear here.

**Comment by vendethiel** [comment]
@geoffliu ?

**Comment by geoffliu** [comment]
👍  Thanks!

**Comment by vendethiel** [comment]
Nice!

**Pull request merged**